extensions-core/lookups-cached-global/src/main/java/io/druid/data/input/MapPopulator.java

```diff
@@ -64,8 +64,11 @@ public long populate(final ByteSource source, final Map<K, V> map) throws IOExce
           @Override
           public boolean processLine(String line) throws IOException
           {
-            map.putAll(parser.parse(line));
-            ++count;
+            final Map<K, V> kvMap = parser.parse(line);
+            if (kvMap != null) {
+              map.putAll(kvMap);
```

Comment:
Not all maps can accept null values. This retains prior functionality of "potentially failing if target map can't handle null values", but I just wanted to make that callout here.